(1) Community Patch/LUA/Includes/TechButtonInclude.lua

```diff
@@ -716,3 +716,44 @@ function AddCallbackToSmallButtons(buttonStack, iButtonCount, nVoid1, nVoid2, eM
 		buttonStack[strButtonName]:RegisterCallback(eMouse, CallbackFunc);
 	end
 end
+
```

Comment:
This was copied without any changes from `DLC\Expansion\Scenarios\SteampunkScenario\TechButtonInclude.lua`